parts/dcosagentvars.t

```diff
@@ -26,4 +26,5 @@
     "{{.Name}}LbIPConfigID": "[concat(variables('{{.Name}}LbID'),'/frontendIPConfigurations/', variables('{{.Name}}LbIPConfigName'))]", 
     "{{.Name}}LbIPConfigName": "[concat(variables('orchestratorName'), '-{{.Name}}-', variables('nameSuffix'))]", 
     "{{.Name}}LbName": "[concat(variables('orchestratorName'), '-{{.Name}}-', variables('nameSuffix'))]",
-{{end}}
\ No newline at end of file
+{{end}}
+    "dcosBootstrapURL": "[parameters('dcosBootstrapURL')]", 
```

Comment:
I'm confused how this works though. ARM allows you to have two variables with the same name in a template? Or am I still missing a detail?